ArduCopter/Log.cpp

```diff
@@ -451,6 +451,15 @@ void Copter::Log_Write_GuidedTarget(uint8_t target_type, const Vector3f& pos_tar
 // units and "Format characters" for field type information
 const struct LogStructure Copter::log_structure[] = {
     LOG_COMMON_STRUCTURES,
+    
+// @LoggerMessage: PTUN
+// @Description: Parameter Tuning information
+// @Field: TimeUS: Microseconds since system startup
+// @Field: Param: Parameter being tuned
+// @Field: TunVal: Normalized value used inside tuning() function
+// @Field: TunMin: Tuning minimum value
```

Comment:
I believe these are specified limits rather than the max/min they've been during tuning, which would be an alternate way of interpreting these descriptions.  Perhaps `Maximum allowed value` or `Tuning maximum limit`?